pkg/standalone/standalone.go

```diff
@@ -585,27 +617,30 @@ func runSchedulerService(wg *sync.WaitGroup, errorChan chan<- error, info initIn
 	if isAirGapInit {
 		// if --from-dir flag is given load the image details from the installer-bundle.
 		dir := path_filepath.Join(info.fromDir, *info.bundleDet.ImageSubDir)
-		image = info.bundleDet.getSchedulerImageName()
-		err = loadContainer(dir, info.bundleDet.getSchedulerImageFileName(), info.containerRuntime)
+		image = info.bundleDet.getDaprImageName()
+		err = loadContainer(dir, info.bundleDet.getDaprImageFileName(), info.containerRuntime)
 		if err != nil {
 			errorChan <- err
 			return
 		}
 	} else {
 		// otherwise load the image from the specified repository.
-		image, err = getSchedulerImageName(imgInfo, info)
+		image, err = getDaprImageName(imgInfo, info)
 		if err != nil {
 			errorChan <- err
 			return
 		}
 	}
 
+	// instanceID is 0 because we run one instance only for now.
+	schedulerDataDir := getSchedulerDataPath(info.installDir, 0)
```

Comment:
design decision: use `~/.dapr/scheduler/data/0` to be future proof the support of multiple instances. Also, it matches the directory name nicely.

Reply:
The data dir in dapr is:
```	
	config.Dir = opts.DataDir + "-" + security.CurrentNamespace() + "-" + opts.EtcdID
```
do we have a concept of a namespace here with the cli that we should acct for too?